cds_ils/documents/api.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2019-2020 CERN.
+#
+# CDS-ILS is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+
+"""CDS-ILS document API."""
+
+from invenio_app_ils.documents.api import Document as ILSDocument
+from invenio_pidstore.resolver import Resolver
+
+DOCUMENT_LEGACY_PID_TYPE = "lrecid"
+
+
+class Document(ILSDocument):
```

Comment:
I think we should not have DOCUMENT_LEGACY_PID_TYPE and SERIES_LEGACY_PID_TYPE because they are identical.

We should probably not override `Document` and `Series` to add the method `get_record_by_legacy_recid`, but put that method as static util somewhere, probably in `literature/api.py`?:
```
LEGACY_PID_TYPE = "lrecid"

    def get_record_by_legacy_recid(cls, pid_value):
        """Get ils record by pid value and pid type."""
        resolver = Resolver(
            pid_type=LEGACY_PID_TYPE,
            object_type="rec",
            getter=cls.get_record,
        )
        _, record = resolver.resolve(str(pid_value))
        return record
```